app/controllers/ctc/questions/cell_phone_number_controller.rb

```diff
@@ -5,32 +5,18 @@ class CellPhoneNumberController < QuestionsController
 
       layout "intake"
 
-      def update
-        super
-      end
-
       private
 
-      def after_update_success
-        RequestVerificationCodeTextMessageJob.perform_later(
-          phone_number: @form.sms_phone_number,
-          locale: I18n.locale,
-          visitor_id: current_intake.visitor_id,
-          client_id: current_intake.client_id,
-          service_type: :ctc
-        )
+      def illustration_path
+        "phone-number.svg"
       end
 
       def prev_path
         questions_contact_preference_path
       end
 
       def next_path
```

Comment:
haha yeah i went back and forth on the removal of this here BECAUSE it is somewhat weird that if you reload the verification page it will send you another text with the code. But we also allow 5 valid codes at once, so if it becomes a problem we can think about it again.